astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 1ecac98d547729393ca54bcbe4f03cd4e1bfcd9a
+Subproject commit cca08a1755e703f9ff94e1f60a5b15d9521c128f
```

Comment:
Yes, do ``git submodule update`` then add the change to the branch. Then later on we can always squash the commits if we want.